src/jsonld/dataset.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
+import Head from 'next/head';
+
+import markup from '../utils/markup';
+
+export interface DatasetJsonLdProps {
+  description: string;
+  name: string;
+  license?: string;
+}
+
+const DatasetJsonLd: FC<DatasetJsonLdProps> = ({
+  description,
+  name,
+  license,
+}) => {
+  const jslonld = `{
+    "@context": "http://schema.org",
+    "@type": "Dataset",
+    "description": "${description}",
+    "name": "${name}"${
+    license
```

Comment:
At the moment if `license` is added it will be invalid JSON due to no trailing comma on `name`.

To fix this if you put `name` last and then update license to always have trailing comma like so: `"license": "${license}",`

Reply:
Hi!
I will actually be a comma on name if license is provided.
See line 23.
The result will be like this:
{
    "@context": "http://schema.org",
    "@type": "Dataset",
    "description": "The description needs to be at least 50 characters long",
    "name": "name of the dataset",
        "license": "https//www.example.com"
  }
The solution is the same to providerUrl in course component.

Reply:
😅 Whoops missed that. Perfect 👌